example-esp32.yaml

```diff
@@ -12,6 +12,8 @@ packages:
     # Change this file to the correct one for your device
     file: econet_heatpump_water_heater.yaml
     # file: econet_hvac.yaml
+    # Use this if you have a comminicating outdoor unit (ODU)
+    # files: [econet_hvac.yaml, econet_hvac_odu.yaml]
```

Comment:
Rather than including this in all three example files, would it be better to include it form the `econet_hvac.yaml`? I don't know whether it should be enable or disabled by default, do you have a sense of that?